build.gradle.kts

```diff
@@ -129,7 +129,7 @@ subprojects {
     extra["logbackVersion"] = "1.2.3"
     extra["mavenVersion"] = "3.5.0"
     extra["ownerVersion"] = "1.0.12"
-    extra["pmeVersion"] = "4.3"
+    extra["pmeVersion"] = "4.4-SNAPSHOT"
```

Comment:
Does this need a 4.4. release before merging?

Reply:
We'll need a release before GA but we can use snapshot dependencies in a snapshot version I think.